client/app/models/landuse-form.js

```diff
@@ -111,6 +112,12 @@ export default class LanduseFormModel extends Model {
     );
   }
 
+  async saveDirtyProject() {
+    if (this.isProjectDirty) {
+      this.package.project.save();
+    }
+  }
```

Comment:
for later: we have a lot of shared behavior across form models. maybe we can make an abstract form model, and these forms just extend from it?

`class LandUseForm extends Form {}`

Reply:
yes, good point!